taxi/models.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+
+    def __str__(self):
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=20, unique=True)
+
+    def __str__(self):
+        return self.username
+
+    class Meta:

```

Comment:
The `verbose_name` and `verbose_name_plural` are missing in your `Meta` class. According to the checklist, they should be in lowercase.